examples/noise-rtc/index.js

```diff
@@ -288,25 +305,29 @@ async function buildWakuMessage(node, noiseExecute) {
 
   const sendMessage = async (message) => {
     let payload = ProtoMessage.create({
-      data: utils.utf8ToBytes(JSON.stringify(message)),
+      //   data: utils.utf8ToBytes(JSON.stringify(message)),
+      data: JSON.stringify(message),
     });
     payload = ProtoMessage.encode(payload).finish();
 
     return node.lightPush.push(encoder, { payload });
   };
 
   const listenToMessages = async (fn) => {
-    return node.filter.subscribe([decoder], (payload) => {
+    return node.filter.subscribe([decoder], ({ payload }) => {
       const { data } = ProtoMessage.decode(payload);
-      fn(JSON.parse(utils.bytesToUtf8(data)));
+      //   fn(JSON.parse(utils.bytesToUtf8(data)));
+      fn(JSON.parse(data));
     });
   };
 
   return [sendMessage, listenToMessages];
 }
 
 function initRTC({ ui, onReceive }) {
-  const configuration = {};
+  const configuration = {
+    iceServers: [{ urls: "stun:stun.l.google.com:19302" }],
```

Comment:
I don;t quite remember the reason for ice and stun. is it when a direct communication cannot be achieved? Can we just skip this? Relying on a google server makes the whole attempt moot.

Reply:
Few slides in https://community.cisco.com/t5/collaboration-knowledge-base/stun-turn-and-ice-for-nat-traversal-in-depth-lecture/ta-p/4437566?attachment-id=203878

STUN seems to be used to enable a device to know its own external address.

This should be achieved already thanks to the libp2p `identify` protocol, during which I expect a remote peer to provide the observed address: https://github.com/libp2p/specs/tree/master/identify#the-identify-message

Reply:
Thank you for pointing it out! I actually was researching how to approach it and didn't notice this options.

Reply:
At this point example is `ready to use` with consideration to `STUN` approach with Google's server.
The next step is to make it work in a decentralised way and one of the approaches will be `Identity` protocol. 

For now I tried to make it work with `libp2p` as a way to get initial offer to initiate RTC connection but it doesn't work. I still need to play with it.

As a `show case` we can merge this example with additional follow-ups to it and our overall approach to `WebRTC` + `Noise`.

I would like to point out two main things about these next steps: ability to exchange SDP info in a correct way and proper abstractions (probably placed in the same place as `js-noise` or separate package). As an additional point I would mention some implementation of `signalling server(TURN)` that wouldn't ruin whole purpose of this effort.

Reply:
It seems we cannot avoid usage of STUN / TURN servers, at lest I cannot find a work around for now. Good explanation to it can be found in this spec 
https://github.com/libp2p/specs/pull/497/files#diff-2cb0b0dcc282bd123b21f5a0610e8a01b516fc453b95c655cf7e16734f2f7b11R48-R53